### DISCUSSION TITLE
🖐️command lines to run on windows 10

### DISCUSSION THREAD

**Discussion by victore447** [trix7777/m2strm#14] [General]
**Good morning,**![143](https://github.com/trix7777/m2strm/assets/48101775/c3b90f67-87e8-41ce-9a29-822bd465ae70)

I am under windows 10 at this step in command line
C:\Users\myuserfolder\Downloads>m2strm.exe /?
C:\Users\myuserfolder\Downloads\m2strm.exe.config
**Afterwards I no longer understand your guide.**
The m2strm.exe.config file has been created in my download folder (see image).![46](https://github.com/trix7777/m2strm/assets/48101775/4838176b-20fc-46c7-98a7-26ae6ade3485)

![strm](https://github.com/trix7777/m2strm/assets/48101775/2824c823-04de-4728-b702-d1d99bccb336)

I have a m3u file downloaded via my iptv subscription in the download folder which I renamed to m3u8.
The tv groups have been manually deleted with software such as iptv editor...
 

What is the command lines to run after to extract
the m3u8 file in strm in my download folder or via the url of my
iptv subscription?

**thank you for the answer**![131](https://github.com/trix7777/m2strm/assets/48101775/7d3063a7-ebd3-4e48-9969-c06f744a545e)

**Comment by trix7777** [comment]
Did you even read the README.md?

...
Example usage:
m2strm.exe original.m3u8
...
Also, you need to edit the m2strm.exe.config, it's a text file with the configuration options in it, described here:
https://github.com/trix7777/m2strm/

**Reply by victore447** [reply]
🖐️
**Precisely I read the README.md and do not understand anything. It is necessary to mention the concrait with the C, CD 
etc.. followed by your examples etc.. It is for this reason that I contact you.**

Here is the result of the command prompt when I rename my m3u8 file by original.m3u8 during the execution 
request that you indicate to me.

**Windows says: 16-bit application not supported with 64-bit, 
contact software vendor for 64-bit version.**

![Incompatible windows64](https://github.com/trix7777/m2strm/assets/48101775/13976600-9803-4f53-8540-295ecd27db96)

**Reply by trix7777** [reply]
You don't need to rename the file, just enter the name of your file in the command instead. Seems you have a faulty exe, where did you download it from? This is the latest release:
https://github.com/trix7777/m2strm/releases/download/v3.0.2.3/m2strm.exe

**Reply by trix7777** [reply]
I'm just guessing here, but did you rename the m2strm.exe.config to m2strm.exe??? Seems you did something to the exe as it was working for you when it created the config-file.

**Comment by trix7777** [comment]
```
D:\source\test>m2strm.exe /G

Quick guide:

1. Create the configuration file: m2strm.exe /C
2. Edit the configuration file and set at least the following:
 - m3u8File value: /where/you/want/to/download/original.m3u8
 - DownloadM3U8Enabled value: True
 - Also fill in your UserURL, UserPort, UserName and UserPass.
3. Download your M3U8-file using the configuration file settings: m2strm.exe /M
4. Create the unwanted groups file: m2strm.exe /U
5. Edit the unwanted groups file and comment out the groups you want to keep with //.
6. You're all done. From now on you can run m2strm.exe without any arguments and the program will be using your configuration.
```

**Reply by trix7777** [reply]
But in your case you seem be wanting to use an already downloaded m3u8-file so you would do this:
(for now, put the exe file and the m3u8-file in a short path eg. C:\test\ and open a cmd window and cd C:\test\)
```
1. Create the configuration file: m2strm.exe /C
2. Edit the configuration file and set at least the following:
<add key="m3u8File" value="C:\test\tv_channels_PAYP5896_plus.m3u" />
keep as is:
<add key="DownloadM3U8Enabled" value="False" />
3. Run m2strm.exe without any arguments and the program will be using your configuration.
```

Alternatively, in your case you don't even need to creat the config file, just run the command:
`m2strm.exe tv_channels_PAYP5896_plus.m3u`

**Reply by victore447** [reply]
👋**Hello, 
First of all thank you for your answers.**

☝️After downloading your exe file on this link 👉 https://github.com/trix7777/m2strm/releases/download/v3.0.2.3/m2strm.exe                                                                                                                         
J managed to extract my m3u8 file with these three commands                                                                                                                                                                                                                                          
CD C:\Users\victo\Downloads
CD C:\User\Victo\Downloads\m2strm.exe /?
m2strm.exe VodKing.m3u8  
**screenshot attached**.![46](https://github.com/trix7777/m2strm/assets/48101775/35283c5d-b6dc-4ac5-a8ef-765837ff39ff)

![Capture Ecran commande M2STRM](https://github.com/trix7777/m2strm/assets/48101775/fdffc026-ffc1-4c27-b832-2eda986ba1ef)

👆👌It was fast and in two minutes for about 9000 movies and 800 tv series that I had edited
via my M3u subscription with a group editor app.
☝️For your first m2strm.exe file which you mentioned
to be defective is on this link that I downloaded it dating from 31/12.21 👉https://github.com/trix7777/m2strm/releases .
And m2strm.exe.config I don't hadn't renamed it to m2strm.exe. I only needed that to convert my m3u8s and will stop
there for the moment because afterwards I'm afraid I won't be able to create the file from the command line.
 unwanted groups etc.... and will try that a bit later.

![22645-applause01](https://github.com/trix7777/m2strm/assets/48101775/a5520a5b-297e-441b-818b-e3f203b0b23e) **Thank you**

**Reply by trix7777** [reply]
That link point to exactly the same exe file as I gave you above. There must have been an error on your download.
If you use this app properly, there's no need to either pre-download or edit the m3u8 for groups as that would be taken care of by the app when configured.

**Reply by victore447** [reply]
Ok I will try that later. Thanks✔️
